From f2f995ebc4c362aaf74679fe37ad66efb6af031d Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Tue, 13 Oct 2015 15:53:38 +0800
Subject: [PATCH 0562/5242] MLK-11701-2 ARM: dts: imx6sx-sdb: add dts support
 for pxp v4l2 output

commit  feb1f1aad7794b4c6aaa231b366a4bbd2611bb02 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add dts support for pxp v4l2 output on imx6sx sdb
board.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sdb.dtsi |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-sdb.dtsi b/arch/arm/boot/dts/imx6sx-sdb.dtsi
index 3499cca..8d4afcd 100644
--- a/arch/arm/boot/dts/imx6sx-sdb.dtsi
+++ b/arch/arm/boot/dts/imx6sx-sdb.dtsi
@@ -65,6 +65,12 @@
 		};
 	};
 
+        pxp_v4l2_out {
+                compatible = "fsl,imx6sx-pxp-v4l2", "fsl,imx6sl-pxp-v4l2";
+                status = "okay";
+        };
+
+
 	vcc_sd3: regulator-vcc-sd3 {
 		compatible = "regulator-fixed";
 		pinctrl-names = "default";
-- 
1.7.9.5

